cluster/examples/kubernetes/rook-cluster.yaml

```diff
@@ -71,6 +84,15 @@ spec:
 #    - name: "172.17.4.101"
 #      directories:         # specific directores to use for storage can be specified for each node
 #      - path: "/rook/storage-dir"
+# you can override the cluster wide resource requests/limits per node, but only
+# when using `useAllDevices: false`!
```

Comment:
actually, should this be `useAllNodes: false`?

Reply:
Fixed it.